lib/Onyx.js

```diff
@@ -614,7 +618,22 @@ function disconnect(connectionID, keyToRemoveFromEvictionBlocklist) {
  */
 // eslint-disable-next-line rulesdir/no-negated-variables
 function notifySubscribersOnNextTick(key, value) {
-    Promise.resolve().then(() => keyChanged(key, value));
+    pendingUpdates.push([key, value]);
+    if (!scheduled) {
+        scheduled = true;
+        setTimeout(() => {
+            scheduled = false;
+            const updatesCopy = pendingUpdates;
+            pendingUpdates = [];
```

Comment:
Maybe the naming is wrong. I basically don't want to add updates while flushing. And leave them for next frame (next flush)

Reply:
Oh, I like that better with the naming, so let's make that change 👍 

I think it's different than what I was concerned about. After having tested this in the JS console I am redacting my original concern about `updatesCopy` being a reference to `pendingUpdates`.

```
let pendingUpdates = [1];
const currentBatch = pendingUpdates;
pendingUpdates = [];
console.log(currentBatch); // Outputs => [1] which is OK. I assumed it would output [].
```